man/org.freedesktop.systemd1.xml

```diff
@@ -1898,6 +1898,8 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       readonly s CollectMode = '...';
       @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
       readonly as Refs = ['...', ...];
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly a(ss) ActivationEventInfo = [...];
```

Comment:
Somehow the "Info" suffix looks weird as a dbus property to me now. Maybe call this ActivationDetails? both in the dbus property and the structure behind it?